substrate/client/network/src/protocol/notifications/handler.rs

```diff
@@ -329,6 +329,9 @@ pub enum NotifsHandlerOut {
 	CloseDesired {
```

Comment:
May be also update the comment above to reflect that `CloseDesired` is emitted when the remote either wishes to close the connection or misbehaves?